scripts/us_census/acs5yr/subject_tables/subject_table_test.py

```diff
@@ -77,6 +78,13 @@ def _read_files(test_path, expected_path):
     return test_result, expected_result
 
 
+def _test_csv_files(test_path, expected_path, precision=3):
+    test_df = pd.read_csv(test_path).round(precision)
+    expected_df = pd.read_csv(expected_path).round(precision)
+
+    return test_df.equals(expected_df)
```

Comment:
In case equals returns false, is there  away to list out diffs as well?

Reply:
Made a small change in the latest commit.
Using [assert_frame_equal](https://pandas.pydata.org/docs/reference/api/pandas.testing.assert_frame_equal.html) now. This will throw an assertion error and points out the column where the two dataframes differ.